src/test/java/org/prebid/server/auction/AuctionRequestFactoryTest.java

```diff
@@ -681,11 +682,35 @@ public void shouldReturnAuctionContextWithTimeout() {
     }
 
     @Test
-    public void shouldReturnAuctionContextWithAccount() {
```

Comment:
Please don't remove the test, just add `verify(applicationSettings).getAccountById(eq("accountId"), any());` for verification.